src/others/cpp_tips.md

```diff
@@ -0,0 +1,27 @@
+---
+tags:
+  - Original
+---
+
+# C++ Tips and Tricks for Competition Programming
+
+## Faster I/O
+
+- `std::ios_base::sync_with_stdio(false)`: by default, C++ works with C++ iostreams (like `std::cin`) and C style stdio streams (like `stdin`) by synchronizing them.
+Since you should never need C style I/O (like `scanf`, `printf`), in C++, disabling this synchronization will make C++ I/O competitive with C style `scanf`.
```

Comment:
Is this also true for `std::print` and `std::println` from C++23? It seems that they're tied into stdout, rather than the cout ostream, and are somewhat more convenient to use. But I don't really know how they fare in terms of performance.

Reply:
I don't know. We can mention it but it's very new and not familiar to most users.